### PR TITLE
Fix default component collection form, and update props field

### PR DESCRIPTION
**Pull request by Asgator**

### CODE REVIEW

File: src/components/collections/form/field.jsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import classNames from 'classnames';
 
 function getClasses(props) {
     let classes = {
-        disabled: props.state === 'disabled',
-        error: props.state === 'error',
```
Comment (asvetliakov) [inline src/components/collections/form/field.jsx]: What is the point to have 'disabled' and 'error' rather than single state="disabled" ?

Comment (Asgator) [inline src/components/collections/form/field.jsx]: Oh, dat fail

**Comment by Asgator** [comment]
@asvetliakov, i remove change in file ```field```.

**Comment by asvetliakov** [comment]
Great, thank you

**Pull request merged**